extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -196,6 +240,25 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
+
+	getParent(element?: BookTreeItem): vscode.ProviderResult<BookTreeItem> {
+		let parentPath;
+		if (element.root.endsWith('.md')) {
+			parentPath = path.join(this._bookPath, 'content', 'readme.md');
+			if (parentPath === element.root) {
```

Comment:
You're always returning the root readme.md as the parent for md files? That doesn't seem right for a couple of reasons :

1. What about md files that aren't at the root?
2. Why is the readme the parent of these? The readme is at the root level but it doesn't have any children

Reply:
I discussed this with both Stuart and @yualan. Looking at the physical folder isn't the right way here. Since in the books explorer, we have the readme file as the parent node and below which we show all the ipynb files.
As for all the readme files inside folders, the root would be the readme inside content.